#! /usr/bin/env python

PACKAGE='jsk_perception'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("degree", double_t,    0, "the size of the degree", 60, 10, 80)
gen.add("scale", double_t,    0, "the size of the output image", 0.2, 0.01, 3)
gen.add("roll", double_t,    0, "roll", 0.78, -1.57, 1.57)
gen.add("pitch", double_t,    0, "pitch", 0.78, -1.57, 1.57)
gen.add("circle_x", double_t,    0, "circle_x", 640, 0, 1280)
gen.add("circle_y", double_t,    0, "circle_y", 512, 0, 1024)
gen.add("circle_r", double_t,    0, "circle_r", 500, 0, 600)

exit(gen.generate(PACKAGE, "fisheye", "Fisheye"))
